src/cat.jl

```diff
@@ -45,17 +43,18 @@ for (T, S) in [
     end
 end
 
-for (f, nf, tf, tup) in [
-    (:vcat, :_named_vcat, :_typed_vcat, ()),
-    (:hcat, :_named_hcat, :_typed_hcat, (:_,)),
-    ]
+for (f, nf, tf, tup) in
+    [(:vcat, :_named_vcat, :_typed_vcat, ()), (:hcat, :_named_hcat, :_typed_hcat, (:_,))]
```

Comment:
I think this was clearer before

Reply:
🤷

Reply:
i agree. And this is the downside of a formatter, even though you can encode a lot of subtle rules, it's hard to write one that captures the subtley of this particular case. We could fix this one up, but it'd change again if we re-ran the formatter so i'm inclined to leave it